src/callbacks/callbacks.jl

```diff
@@ -157,6 +158,17 @@ NVTX.@annotate function rrtmgp_model_callback!(integrator)
         @. ᶜfrac = cloud_diagnostics_tuple.cf
     end
 
+    if !(radiation_mode isa RRTMGPI.GrayRadiation)
+        if radiation_mode.aerosol_radiation
+            ᶜΔz = Fields.local_geometry_field(Y.c).∂x∂ξ.components.data.:9
+            ᶜaero_conc = Fields.array2field(
+                radiation_model.center_aerosol_column_mass_density,
+                axes(Y.c),
+            )
+            @. ᶜaero_conc = p.tracers.prescribed_aerosol_fields.:SO4 * ᶜΔz
+        end
```

Comment:
Will this fail if SO4 is not added in the config? We could add a check and error if the aerosol configuration is not properly set.

Reply:
Yes, I was thinking about what is the best way to do that. Maybe I will add an assert for aerosol_radiation and prescribed_aerosols. I will make some changes shortly and let you take a look again.